pkg/logutil/rate.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package logutil
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pingcap/br/pkg/lightning/metric"
+	"github.com/pingcap/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+)
+
+// TrivialRater is a trivial rate tracer.
+// It doesn't record any time sequence, and always
+// return the average speed over all the time.
+type TrivialRater struct {
+	start   time.Time
+	current prometheus.Counter
+}
+
+// NewTrivialRater make a trivial rater.
+func NewTrivialRater() TrivialRater {
+	return TrivialRater{
+		start: time.Now(),
+		// TODO: when br decided to introduce Prometheus, move this to its metric package.
+		current: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: "BR",
+			Name:      "table_created",
+			Help:      "The count of tables have been created.",
+		}),
```

Comment:
😱 

the `NewTrivialRater` should take a `prometheus.Counter` as input, not create a `table_created` counter every time it is initialized.

Reply:
done, now `RateTracer` become a wrapper over `prometheus.Counter`